### ISSUE TITLE
input-container: not showing ng-messages error by default even when md-auto-hide="false" is applied

### ISSUE THREAD

**Issue by vinhnghi223** [angular/material#9723]
**Actual Behavior**:
-  `What is the issue? *`: Under `md-input-container`, when applied `md-auto-hide="false"` to`ng-messages`, error message(s) is not shown by default.
-  `What is the expected behavior?`: If the input is invalid, error message(s) should be shown by default when `md-auto-hide="false"` is applied on `ng-messages`.

**CodePen** (or steps to reproduce the issue): *
-  `CodePen Demo which shows your issue:` http://codepen.io/vinhnghi223/pen/JRyoXJ ( I used the same example from Angular Material Input Errors demo: https://material.angularjs.org/latest/demo/input) (The `Hourly Rate (USD)` example)
-  `Details:` I realized that the reason why the example on Angular Material Input Errors demo works is each tag, where `ng-message` resides, has an inline styles of  `style="opacity: 1; margin-top: 0px;"`.

**Angular Versions**: *
-  `Angular Version:` 1.5.5
-  `Angular Material Version:` 1.1.1

**Additional Information**:
-  `Browser Type: *` Any
-  `Browser Version: *` Any
-  `OS: *` Any
-  `Stack Traces:` Any

**Comment by vinhnghi223** [comment]
Currently my quick and dirty quick in our project is:

Replace these lines of code:
https://github.com/angular/material/blob/master/src/components/input/input.scss#L262-L268

with these lines of code:

```
&.md-input-invalid,
  &.md-input-has-value,
  &.md-input-focused,
  &.nd-input-focused {
    .md-input-message-animation {
      &:not(.ng-animate) {
        opacity: 0;
        margin-top: -100px;
      }
    }
  }
```

That way the error message will then be visible from the start. So it mitigates the issue. Still the label is colored into red. I guess to make that possible some logic inside `inputTextareaDirective `should be changed.

**Comment by mikila85** [comment]
+1

**Comment by bruno-serfe** [comment]
Hi there, Is this still an issue?

**Comment by Splaktar** [comment]
Fixed in `1.1.5` via https://github.com/angular/material/commit/0151b4bed9d90aa942734591615878f5fcc0fe52.
